_pages/events.md

```diff
@@ -7,7 +7,10 @@ permalink: /about/events/index.html
 
 ## Conferences
 
-### Surveillance Studies Network / Surveillance & Society Biannual Conference (June 2018)
+### Resisting the Chicago Heat List (September 27, 2018)
```

Comment:
NIT: I though we are only adding the month `Sep` and not the date.

Reply:
Ah yes. One sec